src/main/java/thePackmaster/cards/bladestormpack/FeedingWinds.java

```diff
@@ -0,0 +1,105 @@
+import thePackmaster.actions.madsciencepack.FindCardForAddModifierAction;
+import thePackmaster.cardmodifiers.bladestormpack.GainStrengthModifier;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.cards.bladestormpack.FlavorConstants.*;
+import static thePackmaster.util.Wiz.atb;
+import static thePackmaster.util.Wiz.att;
+
+/*REFS: Transmogrifier (pickthemall), Master's Strike (upgradespack), Fuel & ConsumeToDoAction (fueledpack)
+Winterize & FrostOrbModifier & FindCardForAddModifierAction (madsciencepack), AbstractPickThemAllCard(pickthemallpack)*/
+public class FeedingWinds extends AbstractBladeStormCard implements StartupCard, OnCreateCardInterface {
+    public final static String ID = makeID("FeedingWinds");
+    private static final int COST = 0;
+    private static final int CARDS_TO_EXHAUST = 1;  //Could be 2 (not "up to 2") if nerfs are needed.
+    private static final int ENERGY_GAIN = 1;
+    private static final int WINDRUSH = 1;
+    private static final int UPG_WINDRUSH = 1;
+    private static final int STRENGTH_GAIN = 1;
+    private static final int STRENGTH_CARDS = 2;
+    private static final int UPG_STRENGTH_CARDS = 1;
+
+    public boolean combatVersion;
+
+    public FeedingWinds() {
+        super(ID, COST, CardType.SKILL, CardRarity.UNCOMMON, AbstractCard.CardTarget.SELF);
+        baseMagicNumber = magicNumber = STRENGTH_CARDS;
+        baseSecondMagic = secondMagic = WINDRUSH;
+
+        this.combatVersion = false;
+        this.rawDescription = getDescriptionWithStartup();
+        initializeDescription();
+
+        FlavorText.AbstractCardFlavorFields.flavorBoxType.set(this, FLAVOR_BOX_TYPE);
+        FlavorText.AbstractCardFlavorFields.boxColor.set(this, FLAVOR_BOX_COLOR);
+        FlavorText.AbstractCardFlavorFields.textColor.set(this, FLAVOR_TEXT_COLOR);
+    }
+
+    @Override
+    public boolean atBattleStartPreDraw() {
+        this.addToBot(
+            new FindCardForAddModifierAction(
+                new GainStrengthModifier(STRENGTH_GAIN), this.magicNumber, true,
+                AbstractDungeon.player.drawPile, (card) -> card.type == CardType.ATTACK
+            )
+        );
+        return true;    //"true" plays the Startup animation with the card when combat starts.
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        atb(new FeedingWindsAction(CARDS_TO_EXHAUST, secondMagic,
+                new AbstractGameAction() {
+                @Override
+                public void update() {
+                    att(new GainEnergyAction(ENERGY_GAIN));
+                    isDone = true;
+                }
+        }
+        ));
+    }
+
+    @Override
+    public void upp() {
+        upgradeMagicNumber(UPG_STRENGTH_CARDS);
+        upgradeSecondMagic(UPG_WINDRUSH);
+    }
+
+    //From here down, dynamic Startup tooltip based on AbstractPickThemAllCard (pickthemallpack).
+
+    public void onCreateCard(AbstractCard c) {
```

Comment:
A small thing, but as a method that you've implementing from an interface, it's best practice to mark this with `@Override` (just like all the other methods that are originally defined somewhere other than this class). Same comment applies to Flying Swords.